src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailedPackageMetadata.cs

```diff
@@ -118,5 +123,15 @@ public DetailedPackageMetadata(PackageSearchMetadataContextInfo serverData, Pack
         private static readonly IReadOnlyList<PackageDependencySetMetadata> NoDependenciesPlaceholder = new PackageDependencySetMetadata[] { new PackageDependencySetMetadata(dependencyGroup: null) };
 
         public string PackagePath { get; set; }
+
+        virtual public async Task<bool?> GetHasReadMe()
```

Comment:
The name of the method `GetHasReadMe` is a bit confusing to me. How about `HasReadMe` is clear because it is clear to the caller that the method returns a boolean value.

Reply:
I agree though I may want to rethink the name for "HasReadMe" all together. 

Right as a bool? I'm taking the meaning to be: 
True: We have a readme for the package
False: We don't have a readme for the package
Null: We can't determine if a readme is available for the package without going to the server. 

Maybe switching it to an enum would be better so it's clear what we're getting? "ReadmeAvailabilityEnum" with "Available", "Unavailable", and "Unknown"?

Reply:
Given that we changed the method name to `TryGetPackageReadMeAsync` due to another comment, I think this thread can be resolved as no action is needed.